tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -20,4 +20,7 @@
     <suppress files=".+org.openhab.binding.yeelight.+" checks="OutsideOfLibExternalLibrariesCheck" />
     <!-- suppress header checks for imported and patched apache commons-io files in logreader binding -->
     <suppress files=".+org.openhab.binding.logreader.internal.thirdparty.commonsio.+" checks="ParameterizedRegexpHeaderCheck|AuthorTagCheck" />
+	<!-- suppress author tag checks for generated sources in tr064 binding -->
+    <suppress files=".+org.openhab.binding.tr064.internal.dto.+" checks="AuthorTagCheck" />
```

Comment:
See my comment above. I would prefer to have them generated during the build (which is not an additional step but done on-the-fly). This would remove the need to manually add the headers and also remove the need to have the author tag-check-excluded.